static/js/containers/LoginButton.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import DropdownButton from 'react-bootstrap/lib/DropdownButton';
+import { LinkContainer } from 'react-router-bootstrap';
 import MenuItem from 'react-bootstrap/lib/MenuItem';
 import Link from 'react-router/lib/Link';
```

Comment:
`React` is a strange one because every `<div />` gets translated to a `React.createElement("div")` which requires `React`, but you don't see it in the original code. It looks like we can filter out certain errors based on variable name: http://eslint.org/docs/rules/no-unused-vars#caughterrorsignorepattern